Sensors/SMCSuperIO/NuvotonDevice.hpp

```diff
@@ -29,6 +29,19 @@ namespace Nuvoton {
 	static constexpr uint16_t NUVOTON_VBAT_REG 					= 0x488;
 	static constexpr uint16_t NUVOTON_VBAT_6775_REG				= 0x551;
 	static constexpr uint16_t NUVOTON_VBAT_CONTROL_REG			= 0x5D;
+	// NCT6683 specific data
+	static constexpr uint8_t NUVOTON_6683_PAGE_REGISTER_OFFSET	= 0x04;
+	static constexpr uint8_t NUVOTON_6683_INDEX_REGISTER_OFFSET	= 0x05;
+	static constexpr uint8_t NUVOTON_6683_DATA_REGISTER_OFFSET	= 0x06;
+	static constexpr uint8_t NUVOTON_6683_EVENT_REGISTER_OFFSET	= 0x07;
+	static constexpr uint8_t NUVOTON_6683_MON_NUMS				= 32;
+	static constexpr uint16_t NUVOTON_6683_MON_REGISTER_OFFSET	= 0x100;
+	static constexpr uint16_t NUVOTON_6683_MON_CFG_OFFSET		= 0x1A0;
+	static constexpr uint16_t NUVOTON_6683_MON_VOLTAGE_START	= 0x60;
+	static constexpr uint8_t NUVOTON_6683_FAN_NUMS				= 16;
+	static constexpr uint16_t NUVOTON_6683_FAN_REGS[] = { 0x142, 0x140, 0x144, 0x146, 0x148, 0x14A, 0x14C, 0x14E, 0x150, 0x152, 0x154, 0x156, 0x158, 0x15A, 0x15C, 0x15E };
+	static constexpr uint8_t NUVOTON_6683_VOLTAGE_NUMS			= 23;
+	static uint16_t NUVOTON_6683_VOLTAGE_REGS[NUVOTON_6683_VOLTAGE_NUMS] = { 0 };
```

Comment:
I do not like this being a global variable, in my opinion, it should really be part of the NuvotonDevice. It should also not be in upper case as it is not a constant.